Indicators/LinearWeightedMovingAverage.cs

```diff
@@ -77,7 +77,7 @@ protected override decimal ComputeNextValue(IReadOnlyWindow<IndicatorDataPoint>
 
             // The denominator is calculated each time in case the Size is less than the period.
             // There may be a more efficient way of calculating the factorial.
-            for (var i = 0; i <= index; i++)
+            for (var i = 0; i < index; i++)
```

Comment:
why not `var denominator = index`?

Reply:
Hi, if you mean this as a technical suggestion, then I am unable to comment. Your suggestion is most likely better. 

I am only trying to supply the smallest, most compatible change to what I believe may be a mathematical/logical error in the implementation of this indicator.